src/variable.jl

```diff
@@ -130,17 +130,19 @@ mutable struct Configuration{V,P,O}
         if neighbor == []
             # By default, only the order-1 and order+1 diagrams are considered to be the neighbors
             # Nd is the normalization diagram, by default, it only connects to the first diagram
-            neighbor = Vector{Vector{Int}}([[d - 1, d + 1] for d in 1:Nd])
-            neighbor[1] = (Nd == 2 ? [2, ] : [Nd, 2]) # if Nd=2, then 2 must be the normalization diagram
-            neighbor[end] = [1, ] # norm to the first diag
-            (Nd >= 3) && (neighbor[end - 1] = [Nd - 2, ]) # last diag to the second last, possible only for Nd>=3
+            neighbor = Vector{Vector{Int}}([[d - 1, d + 1] for d = 1:Nd])
```

Comment:
what's the difference between 
for d in a:b
and
for d = a:b

Reply:
no difference